data.sql

```diff
@@ -4,3 +4,11 @@ INSERT INTO animals VALUES (1, 'Agumon', TO_DATE('03/02/2020', 'DD/MM/YYYY'), 0,
 INSERT INTO animals VALUES (2, 'Gabumon', TO_DATE('15/11/2018', 'DD/MM/YYYY'), 2, true, 8);
 INSERT INTO animals VALUES (3, 'Pikachu' , TO_DATE('07/01/2021', 'DD/MM/YYYY'), 1, false, 15.04);
 INSERT INTO animals VALUES (4, 'Devimon' , TO_DATE('12/05/2017', 'DD/MM/YYYY'), 5, true, 11);
+
+INSERT INTO animals VALUES (5, 'Charmander' , TO_DATE('08/02/2020', 'DD/MM/YYYY'), 0, false, 11);
+INSERT INTO animals VALUES (6, 'Plantmon' , TO_DATE('15/11/2021', 'DD/MM/YYYY'), 2, true, 5.7);
+INSERT INTO animals VALUES (7, 'Squirtle' , TO_DATE('02/04/1993', 'DD/MM/YYYY'), 3, false, 12.13);
+INSERT INTO animals VALUES (8, 'Angemon' , TO_DATE('12/06/2005', 'DD/MM/YYYY'), 0, true, 45);
```

Comment:
- [ ] The weight values in these queries are negative, Please make sure that you fix the data.

![image](https://user-images.githubusercontent.com/83803414/214351154-cd1e5585-01ab-4b00-9ac7-b358142ffbd4.png)

Reply:
Thanks, I saw it as a hyphen.